modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceconfiguration/update/DatabaseServiceConfigurationUpdateStorage.java

```diff
@@ -31,4 +34,11 @@ public ServiceConfigurationUpdateRequest storeAndFlush(
             ServiceConfigurationUpdateRequest serviceConfigurationUpdateRequest) {
         return serviceConfigurationUpdateRepository.saveAndFlush(serviceConfigurationUpdateRequest);
     }
+
+    @Override
+    public void batchInsert(List<ServiceConfigurationUpdateRequest> requests) {
+        if (!CollectionUtils.isEmpty(requests)) {
+            requests.forEach(serviceConfigurationUpdateRepository::saveAndFlush);
```

Comment:
We can add  a new method `<S extends T> List<S> saveAll(Iterable<S> entities);`  in the `CustomJpaRepository`, and then use the new method `saveAll` to improve efficiency by reducing database connections

Reply:
agreed. I Missed this.